conf/evolutions/default/4.sql

```diff
@@ -2,13 +2,14 @@
 
 CREATE TABLE `patients` (
   `id` INT NOT NULL AUTO_INCREMENT ,
+  `kit_id` INT NOT NULL,
   `first_name` VARCHAR(255) NOT NULL ,
   `last_name` VARCHAR(255) NOT NULL ,
   `age` INT NOT NULL ,
   `sex` VARCHAR(10) NOT NULL ,
   `address` VARCHAR(255) NOT NULL ,
   `city` VARCHAR(255) NOT NULL ,
-  PRIMARY KEY  (`id`) ,
+  PRIMARY KEY  (`id`, `kit_id`) ,
   UNIQUE INDEX `id_UNIQUE` (`id` ASC))
 DEFAULT CHARACTER SET = utf8;
```

Comment:
dont we usually want to create a new evolutions file for db changes? if we were to push this the existing kits would have to rebuild starting at this evolution instead of only adding the new one

Reply:
Ya I believe you are right. These changes happened before we understood evolutions. I can go back and make new sql files.